ruby/phase-1.md

```diff
@@ -0,0 +1,598 @@
+# Ruby SDK - Phase 1
+
+Ruby is a popular dynamic programming language that many companies are leaning on to beat their
+competitors to the market. With the focus on the speed of execution it is very common for these
+companies to rely on managed solutions. This presents an opportunity for Temporal to become Ruby's
+de facto standard for workflow orchestration.
+
+
+## Timeline
+
+The Ruby SDK will be implemented in four loosely defined phases:
+
+- Phase 1 — initial setup, activity APIs, client and worker, payload converters
+- Phase 2 — per-class execution attributes, logging, metrics, error handling, middleware
+- Phase 3 — workflow APIs, local activities, async activity completion, etc
+- Phase 4 — Sandbox, replayer, support for testing
+
+
+## Overview
+
+The Ruby SDK will be implemented on top of the [SDK Core](https://github.com/temporalio/sdk-core).
```

Comment:
Includes a client, yes. As far as I understand it is the suggested approach and I don't see strong reasons not to use it. This is also a decision that can easily be reverted since the interface for either is expected to be very similar (if not identical).